libs/zlog/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=zlog
+PKG_VERSION:=1.2.15
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/HardySimpson/$(PKG_NAME)/archive/refs/tags/
+PKG_HASH:=00037ab8d52772a95d645f1dcfd2c292b7cea326b54e63e219a5b7fdcb7e6508
+PKG_MAINTAINER:=Marko Ratkaj <markoratkaj@gmail.com>
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/zlog
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Pure C logging library
+  URL:=http://hardysimpson.github.io/zlog/
+endef
+
+MAKE_FLAGS+= \
+	PREFIX="$(PKG_INSTALL_DIR)/usr"
+
+define Package/zlog/description
+  zlog is a reliable, high-performance, thread safe, flexible, clear-model, pure C logging library.
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/ $(1)/usr/lib/
+	$(CP) $(PKG_BUILD_DIR)/src/*.h $(1)/usr/include/
+	$(CP) $(PKG_BUILD_DIR)/src/libzlog.a $(1)/usr/lib/
```

Comment:
if PKG_INSTALL is passed, why use PKG_BUILD_DIR here?

Reply:
Thanks for the review. You are right, should use PKG_INSTALL_DIR instead. Will fix..

Reply:
Ah managed to screw it all up with a merge commit.... Will close pull request, squash commits, rebase and re-upload fixed version. Sry and tnx again.